src/d3a/models/area/stats.py

```diff
@@ -157,3 +167,29 @@ def _aggregate_exported_imported_energy(self):
             self.imported_traded_energy_kwh[self.current_market.time_slot] = 0.
         if self.current_market.time_slot not in self.exported_traded_energy_kwh:
             self.exported_traded_energy_kwh[self.current_market.time_slot] = 0.
+
+    def calculate_energy_deviances(self):
+        """
+        If area is a device - Get its forecated deviance
+        Else accumulate energy deviances of connected children
+        """
```

Comment:
Nit, but would be great to convert this to the Google docstring format that we have decided to conform to.